public/js/loader.js

```diff
@@ -25,7 +25,8 @@ class Loader {
    *
    * @param container {HTMLElement} the element containing the loader.
    */
-  constructor(container) {
+  constructor(container, color) {
```

Comment:
I'm wondering if we should pass an extra class to be added instead of a color. Could make it more flexible.